### ISSUE TITLE
[Core] - Sometimes characters can fly as if they were swimming in air.

### ISSUE THREAD

**Issue by Clangeddin** [GuabinaCore/WoWKoi#1218]
**Links:**
https://streamable.com/dkda6f
(video above showing the character flying, first seconds are frozen, video begins after 15 sec or so)

**What is happening:**
Well, you start flying when you hit the jump button, without a mount or anything.
Sadly I havent yet understood how to replicate this.
Relogging fixes the issues (until it happens again, somehow).

**What should happen:**
None of the above.

**Comment by Fhroozt** [comment]
This happens after some pieces of scripted content i.e northern barrens caravan delivery quests. I made a bug report a couple of days ago #1164

**Comment by NightwindOI** [comment]
One way to reproduce it by 100% --> go on the fly mount [Obsidian Nightwing](https://www.wowhead.com/spell=121820/obsidian-nightwing) --->turn/take any quest that includes cinematic-like [Rheastrasza's Gift](https://www.wowhead.com/quest=27898/rheastraszas-gift)  ---> after the cinematic is over=u can fly.

**Comment by taktrix** [comment]
I think I've figured out what the cause of this bug is. It seems to happen when the action you're taking is something that wasn't scripted. For example I was in the middle of a quest where you had to look through a lens to see a certain area. I cancelled in the middle of that animation, because the game didn't know what action to take when I cancelled it popped me out where the lens had gotten to and I was flying. Another example of this was when I was doing an escort caravan quest from far watch post. In the ending sequence I jumped in the middle of it, and since the game didn't know what to do as a default I started to fly. Basically it seems to be a default filler action for when the game is unsure of what to do.
